decidim-proposals/app/helpers/decidim/proposals/application_helper.rb

```diff
@@ -39,22 +40,28 @@ def proposal_state_css_class(state)
         end
       end
 
-      # Public: The css class applied based on the proposal state to
-      #         the proposal badge.
+      # Public: The state of a proposal in a way a human can understand.
       #
       # state - The String state of the proposal.
       #
       # Returns a String.
-      def proposal_state_badge_css_class(state)
```

Comment:
Why did you remove this?

Reply:
It's not used any more as the badge is a `cell` as seen in `proposals/show.html`:
https://github.com/decidim/decidim/blob/f4e87efd187dc6f532a57f6367289628c1463996/decidim-proposals/app/views/decidim/proposals/proposals/show.html.erb#L37

the `proposal_state_badge_css_class` is a legacy method from the `proposals/_proposal_badge.html.erb` partial, which I think is not used anymore, so it could be deleted (?):

https://github.com/decidim/decidim/blob/f4e87efd187dc6f532a57f6367289628c1463996/decidim-proposals/app/views/decidim/proposals/proposals/_proposal_badge.html.erb#L1-L6